aerogear/unifiedpush-operator
# Remove CRDs & make it a namespace operator & add sync period & remove limit on number of UPSes & go back to old UPS version

## Motivation
JIRA: https://issues.jboss.org/browse/AEROGEAR-10091

## What
BURN IT ALL DOWN

## Why
- We don't use those CRDs any longer
- We want to use the older version of UPS
- We don't have any reason to need cluster roles now
- We don't have any reason to limit to a single UPS
- Forcing a sync every 5 minutes sounds like a better idea than waiting for 10 hours or whatever the default is

## How
Deleted some code, updated some other code

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Deploy the operator
2. Deploy a UPS CR
3. Make sure everything works as expected
4. Delete the UPS CR
5. Make sure everything works as expected

## Checklist:
